sdk/cosmosdb/arm-cosmosdb/samples/v16/typescript/package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@azure/arm-cosmosdb": "latest",
     "dotenv": "latest",
-    "@azure/identity": "^4.2.1"
+    "@azure/identity": "^4.0.1"
```

Comment:
is this expected?

Reply:
it should be, don't know why in the old commit it became 4.2.1